docs/add-ins/addin-tutorial.md

```diff
@@ -1,107 +1,219 @@
 ---
-title: Advanced Outlook add-in tutorial
-description: Follow along to create an Outlook add-in that inserts GitHub Gists into the body of new email messages.
-author: jasonjoh
-ms.topic: get-started-article
-ms.technology: office-add-ins
-ms.devlang: javascript
-ms.date: 09/13/2018
-ms.author: jasonjoh
+title: 'Tutorial: Build a message compose Outlook add-in'
+description: In this tutorial, you'll build a Outlook add-in that can be used in message compose mode to insert GitHub gists into the body of a new message.
```

Comment:
Should be "..build an Outlook"
Recommend: In this tutorial, you'll build an Outlook add-in that inserts GitHub gists into the body of a new message.